### ISSUE TITLE
Feature Suggestion : Event : Braking, Acceleration, Turn

### ISSUE THREAD

**Issue by nefertiti007** [tananaev/traccar#2237]
Dear All,

I have implemented this on the front end where I query the position table to identify,. 

Harsh Acceleration, Harsh Braking and Sudden or Fast Turns.

Where for Harsh Acceleration & Braking:,. I check the time taken between two points, and the distance covered, and the difference of speed between both values (Which I have preset) based on those, the two points data is displayed as harsh braking or acceleration.

Similarly, if there is a sudden change of course/direction & speed, configurable,. Its identified as sudden swerve/turn etc.

I think someday down the link, It would be nice to Add these as well :+1:  maybe if there is demand for this feature.

**Comment by tananaev** [comment]
I think those kinds of events should be reported by device. Some devices actually do reports alarms like harsh breaking/acceleration etc.

**Comment by nefertiti007** [comment]
@tananaev yes I do agree, I have also seen some devices do it, but the devices I procure and use do not have them so I just happened to try it on custom php front end :) turned out fairly good when I tested with actual vehicle etc. 

Some love the idea, but am sure its not the most important feature at present :)

**Comment by tananaev** [comment]
OK, thanks for suggestion.

**Comment by tananaev** [comment]
I don't remember off the top of my head.

**Comment by tananaev** [comment]
We already support all those events, but they should be supported on the device side as well.
